elizhahn/overlook
# Feature/filter by date and room type

# Is this a fix or a feature?
-feature
# What is the change?
- The method in the hotel class: filterRooms() was changed so it does not update the available rooms property
- this adds functionality to the page display, a user can now enter a date and see the available rooms displayed for that date
- the user can also filter by room type and see only the rooms of that type displayed
# What does it fix?
# Where should the reviewer start?
- index.js
# How should this be tested?
- run npm test all tests should still be passing
-visit localHost8080 and run through the date and filter button options, they should all be displaying properly
# Screenshots (if appropriate):
